lib/rpc/chains.ts

```diff
@@ -0,0 +1,21 @@
+import { ChainId } from '@uniswap/sdk-core'
+
+export const CHAIN_IDS_TO_NAMES = {
```

Comment:
btw dont we already have such a conversion? otherwise how do we pass in the chain currently?

Reply:
Yea, routing-api imports `ID_TO_NETWORK_NAME` from sor